qemu/tests/cfg/multi_disk.cfg

```diff
@@ -5,7 +5,18 @@
     remove_image = yes
     remove_image_image1 = no
     cmd_timeout = 1000
-    black_list = C:
```

Comment:
no need always both add Linux and Windows,
you can set linux as default setup, and add
Windows: